### ISSUE TITLE
Migration to organization

### ISSUE THREAD

**Issue by Neustradamus** [veeam/veeamsnap#8]
It is possible to migrate the personal account to organization?

Converting:
- https://help.github.com/articles/converting-a-user-into-an-organization/

It is possible to rename this account and create the organization and move/transfer the repository to the organization.

Rename account:
- https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username

Create an organization:
- https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch

Transfer a repository:
- https://help.github.com/en/github/administering-a-repository/transferring-a-repository

Like:
- http://github.com/gnome
- http://github.com/kde
- ...

**Comment by Neustradamus** [comment]
@SergeiShtepa: It has not been solved, can you reopen it?

**Comment by SergeiShtepa** [comment]
Of course, this problem can be solved, but it does not concern the veeamsnap project.
I consider issue as problems of users who use the project.
And such proposals would be appropriate to send a message to the owner of the veeam account.

Best regards.

**Comment by Neustradamus** [comment]
@SergeiShtepa: The ticket is here for @veeam (not for you originally except if you are @veeam too), it is at the good place.
Can you connect with @veeam account?

I can create a ticket in all repositories of course.

It is not good to close a ticket when it is not solved.
Can you reopen?

Thanks in advance.

**Comment by veeam** [comment]
@Neustradamus ,

Answering your question - yes, it is possible to migrate this account to organization.
Why are you asking though?

Thanks!

**Comment by Neustradamus** [comment]
Why I request you?

Because it makes sense, it will be better, ...
Easy management and it is free.

At the same time, please add the avatar/picture and add other informations to be more serious.

Thanks in advance.

**Comment by veeam** [comment]
@Neustradamus

That does not really justify the move and collateral expenses. So far it is not obvious that an organization account is better.

The current personal account also provides easy management and it is also free. So far we are quite happy with what we have now (it just works).

We might be revisiting your proposal later, though.

Speaking of 'add other information' - what kind of information do you think is missing and where?

Thank you

**Comment by Neustradamus** [comment]
Dear @veeam team,

Have you progressed on this unsolved ticket?

Thanks in advance.

**Comment by veeam** [comment]
Hi,

The previous comment still holds true, nothing has changed since then.

Thanks

**Comment by Neustradamus** [comment]
@veeam: It takes few minutes to be in legality...

**Comment by Neustradamus** [comment]
Dear @veeam team,

I comeback to you, we are now in 2025, can you convert the personal account into organization?

Soon 5 years old of this ticket.

GitHub TOS:
- https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements

Personal account is for a HUMAN.

Veeam is a company:
- https://www.veeam.com/
- https://www.veeam.com/company/about.html
©2025 Veeam® Software

Can you convert?

Thanks in advance.

**Comment by veeam** [comment]
Hi,

@Neustradamus,

FYI - we have started internal process of migrating the account.

Thanks

**Comment by Neustradamus** [comment]
Dear @veeam,

It is a very good news! :)

It is really easy to do it, it takes few seconds/minutes.

**Comment by Flole998** [comment]
Looks like the migration has been completed by now :)

**Comment by Neustradamus** [comment]
@Flole998: Nice, thanks!

Can you change the visibility from private to public in team?
- https://github.com/orgs/veeam/people
